modules/cloud4/modules/administration/nav.adoc

```diff
@@ -0,0 +1,18 @@
+* xref:index.adoc[Administration]
+** xref:administration:how2-invite-users.adoc[]
+** xref:administration:how2-access-mgnt.adoc[]
+** xref:administration:security/index.adoc[]
+*** xref:administration:security/idp.adoc[]
+*** xref:administration:security/password-policy.adoc[]
+** xref:administration:billing/index.adoc[Billing]
+*** xref:administration:billing/payment-methods.adoc[]
+*** xref:administration:billing/invoices.adoc[]
+** xref:administration:settings/index.adoc[Settings]
+*** xref:administration:settings/how2-use-organization-mgnt.adoc[]
+*** xref:administration:settings/how2-create-api-key.adoc[]
+
+////
+** xref:cloud4:security:index.adoc[]
+*** xref:cloud4:security:password-policy.adoc[]
+*** xref:cloud4:security:idp.adoc[]
```

Comment:
I noticed you are using cloud4 in the path for these, but not for the others.  A best practice with Antora is to use the most local path that you can. In this case:
* We are in the administration folder, so you could remove **administration** from the upper group
* **Security** is on the same level of **adminstration**. You don't need to go all the away up to **cloud4**.

While this works fine now, if you decide to move or rename folders, it is either zero effort or more clear about what to do, if you use the local paths.

One tip: Never include the component name in the reference if the target and source are in the same component. Mentioning the component name will default to the newest version of the component, which is not what users want for components with multiple versions (such as the server or GSQL).

Reply:
Those has been commented out, I will remove them